mmv1/products/vertexai/FeatureGroup.yaml

```diff
@@ -0,0 +1,125 @@
+update_verb: :PATCH
+update_mask: true
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/vertex-ai/docs'
+  api: 'https://cloud.google.com/vertex-ai/docs/reference/rest/v1/projects.locations.featureGroups'
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+    resource_inside_response: true
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+description: |-
+  Vertex AI feature Group.
+autogen_async: false
+skip_sweeper: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'vertex_ai_featuregroup_basic'
+    primary_resource_id: 'test-tf-featuregroup'
+    primary_resource_name: "fmt.Sprintf(\"terraform%s\",
+      context[\"random_suffix\"\
+      ])"
+    vars:
+      name: 'test_tf_featuregroup'
+      project: 'appeng-flex'
+    test_env_vars:
+      org_id: :ORG_ID
+      billing_account: :BILLING_ACCT
+    ignore_read_extra:
+      - 'force_destroy'
+parameters:
+  - !ruby/object:Api::Type::String
+    name: region
+    description: The region of the dataset. eg us-central1
+    url_param_only: true
+    immutable: true
+    default_from_api: true
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    description:
+      The name of the FeatureGroup. This value may be up to 60 characters, and
+      valid characters are [a-z0-9_]. The first character cannot be a number.
+    immutable: true
+    url_param_only: true
+    pattern: projects/{{project}}/locations/{{region}}/featureGroups/{{name}}
+    custom_flatten: templates/terraform/custom_flatten/name_from_self_link.erb
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      description of the FeatureGroup.
+  - !ruby/object:Api::Type::String
+    name: 'etag'
+    description: Used to perform consistent read-modify-write updates.
+    output: true
+    ignore_read: true
+  - !ruby/object:Api::Type::String
+    name: 'createTime'
+    output: true
+    description: |
+      The timestamp of when the featureGroup was created in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+  - !ruby/object:Api::Type::String
+    name: 'updateTime'
+    output: true
+    description: |
+      The timestamp of when the featureGroup was last updated in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+  - !ruby/object:Api::Type::KeyValueLabels
+    name: 'labels'
+    description: |
+      A set of key/value label pairs to assign to this featureGroup.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'bigQuery'
+    required: true
+    immutable: true
```

Comment:
Just want to confirm that this field is immutable or not? With `immutable: true`, the resource will be recreated when this field is changed.